src/components/BaseButton.vue

```diff
@@ -3,14 +3,11 @@
     class="bg-blue-primary text-white text-center p-4 rounded-md font-bold text-xl shadow-xl shadow-blue-400 mb-4"
   >
     <slot name="icon" />
-    {{ title }}
+    {{ props.title }}
```

Comment:
```suggestion
    {{ title }}
```

The props are accessible directly in the template no need to use the word `props`.